copy-sent-datetime-from-events.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy import create_engine, and_
+from sqlalchemy.orm import sessionmaker
+from secure_message.repository.database import SecureMessage, Events
+from config import Config # NOQA
+
+engine = create_engine(Config.SQLALCHEMY_DATABASE_URI)
+Session = sessionmaker(bind=engine)
+session = Session()
+
+try:
+    messages_without_sent_at = session.query(SecureMessage).filter(SecureMessage.sent_at == None).all() # noqa
```

Comment:
Should this be `(SecureMessage.read_at is None)` as it is complaining about an comparison error

Reply:
Have you tried it yourself?  I think I tried doing it the way they wanted it and I couldn't get it to work.  I think there are other examples of that in the application code for the same reason

Reply:
Yeah I tried this and it doesn't work so I'm happy with whats been decided atm after having a chat with Darrell.

Reply:
Yeah I've tried this with the whole `.filter(SecureMessage.sent_at is None)` and it did not work. I'm happy with the current solution after having a chat with Darrell as well on some clarification.